### PR TITLE
Connection cleanup (lazy_relay)

### PR DESCRIPTION
**Pull request by Raynos**
Another piece of cleanup ahead of lazy relaying.

r: @jcorbin @shannili @kriskowal

### CODE REVIEW

File: node/connection.js [hunk 1]
```diff
@@ -529,14 +534,11 @@ TChannelConnection.prototype.resetAll = function resetAll(err) {
             return;
         }
 
-        var info = {
+        var info = req.extendLogInfo({
```
Comment (ShanniLi) [inline node/connection.js]: Why do you need to add this extra layer of function call?

**Comment by ShanniLi** [comment]
lgtm with a comment.

File: node/connection.js (hunk 1 above)
Comment (Raynos) [inline node/connection.js]: We are going to store lazy requests in the operations in the future.

Lazy requests do not have a bunch of fields so we need branching between eager vs lazy requests.

**Comment by kriskowal** [comment]
:ok:

**Comment by jcorbin** [comment]
:+1:

**Pull request merged**